functions/update/index.js

```diff
@@ -0,0 +1,40 @@
+import { parseAssignedProperties, fetchRecord } from '../utils';
+
+const update = async ({
+  selectedRecord: {
+    data: { id: recordId },
+    model: { name: modelName },
+  },
+  mapping,
+}) => {
+  const assignProperties = parseAssignedProperties(mapping);
+
+  const input = {
+    ...assignProperties,
```

Comment:
Is the `updatedAt` set by the DataAPI?

Reply:
yes it is. I validated that by removing it, publishing it and run it

Reply:
Great! Can you check if `createdAt` and `updatedAt` are also set by the DataAPI in the `create` mutation? And if so, remove those fields?

Reply:
Removed as well